### DISCUSSION TITLE
does not work

### DISCUSSION THREAD

**Discussion by genrihx2** [WilliamVenner/gmpublisher#157] [Help]
does not work  gmpublisher   

![image](https://user-images.githubusercontent.com/3043207/210348902-96b77fb9-a144-45c4-9f50-1f47108bdeb6.png)

**Comment by WilliamVenner** [comment]
You probably need to fix the location of the game in settings

**Comment by genrihx2** [comment]
me fix location   E:\Program Files (x86)\SteamLibrary\steamapps\common\GarrysMod
![image](https://user-images.githubusercontent.com/3043207/210394040-2e21d02c-928f-45e4-bb94-1b04a21a5819.png)

**Comment by genrihx2** [comment]
![image](https://user-images.githubusercontent.com/3043207/210395618-3a703172-f684-49d8-bfba-36b0f97fca18.png)

 error    workshop

**Comment by WilliamVenner** [comment]
Looks like a problem with your Steam then

**Comment by genrihx2** [comment]
yes  problem   Steam

**Comment by Be1zebub** [comment]
i had the same issue some time

**Reply by WilliamVenner** [reply]
how did you resolve it?

**Reply by Be1zebub** [reply]
just wait a bit & got it work - idk
